webserver/webserver/src/main/java/io/helidon/webserver/WebServer.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017, 2018 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2017, 2018, 2019 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Wrong copyright. Should be `2017, 2019`

Reply:
This is actually valid, but not our preferred form. I will fix that.